src/pages/Countries.js

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate, useLocation } from 'react-router-dom';
+import { Navbar, Nav } from 'react-bootstrap';
+import { getCountries } from '../Redux/Country/countriesSlice';
+import Country from '../components/Country';
+import Wrapper from '../components/Wrapper';
+
+const Countries = () => {
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+  const location = useLocation();
+  const searchParams = new URLSearchParams(location.search);
+  const continent = JSON.parse(searchParams.get('continent') || '{}');
```

Comment:
- Hey, we should be able to search for countries by its name by adding a search bar where users can search for countries by their names. this should be implemented using the filter method. kindly add this feature. see the image below on where to add it.

*search requirement*
![Screenshot from 2023-02-17 23-32-14](https://user-images.githubusercontent.com/59397159/219811010-ff1bbe09-218c-4866-b7ac-3fa32dcd924f.png)



![Screenshot from 2023-02-17 23-25-19](https://user-images.githubusercontent.com/59397159/219810379-685fb712-d227-42d9-8ccb-bc1c245fc83d.png)

Reply:
Hi @zieeco This feature has been implemented.

![React App (3)](https://user-images.githubusercontent.com/29442846/219845676-1452a421-ad11-4c0d-9ea2-9f5dcae596b0.png)